package.json

```diff
@@ -86,7 +86,7 @@
     "ts-node": "^10.9.1",
     "typedoc": "^0.23.10",
```

Comment:
Ditto, do these need updates?

Reply:
`ts-node` is already at the latest version, and I'm not sure there needs to be a coupling between the TS and the typedoc updates.